data/migrations/34.lua

```diff
@@ -1,25 +1,8 @@
 function onUpdateDatabase()
 	print("> Updating database to version 34 (re work on outfits(addons) & mount saving/loading)")
-	db.query(
-	--[[
-	CREATE TABLE IF NOT EXISTS `player_outfits` (
-	`player_id` int NOT NULL DEFAULT '0',
-	`outfit_id` smallint unsigned NOT NULL DEFAULT '0',
-	`addons` tinyint unsigned NOT NULL DEFAULT '0',
-	PRIMARY KEY (`player_id`,`key`),
-	FOREIGN KEY (`player_id`) REFERENCES `players`(`id`) ON DELETE CASCADE
-	) ENGINE=InnoDB DEFAULT CHARACTER SET=utf8;
-
-	CREATE TABLE IF NOT EXISTS `player_mounts` (
-	`player_id` int NOT NULL DEFAULT '0',
-	`mount_id` smallint unsigned NOT NULL DEFAULT '0',
-	PRIMARY KEY (`player_id`,`key`),
-	FOREIGN KEY (`player_id`) REFERENCES `players`(`id`) ON DELETE CASCADE
-	) ENGINE=InnoDB DEFAULT CHARACTER SET=utf8;
-
-	ALTER TABLE players
-	ADD currentmount smallint unsigned NOT NULL DEFAULT '0';
-	]])
```

Comment:
It was better to read with the line breaks :|

Reply:
problem is I'm getting errors when I had them in brackets but I'll try to make a better formating